tdrs-backend/gunicorn_development.py

```diff
@@ -0,0 +1,23 @@
+"""Gunicorn development config file."""
+
+# WSGI application path MODULE_NAME:VARIABLE_NAME
+wsgi_app = "tdpservice.wsgi:application"
+
+# The granularity of Error log outputs
+loglevel = "debug"
+# The number of worker processes for handling requests
+workers = 2
```

Comment:
I know we arbitrarily chose 3 workers before but why the downgrade to 2?

Reply:
In fact the number of workers were chosen based on recommendation ```(2 x $num_cores) + 1```. However, there is a loose logic behind this formula and I found 2 workers should work just fine.

Reply:
So did some diving this weekend based on this and some cloudfoundry documentation I stumbled through when supporting 1007. I'm curious if we should indeed use 2 workers but look into auto-scaling if workload increases as that is how I think cloudfoundry is intended to be used.

Reply:
In fact we should revert it back to 3 since one worker is being used for management and the other two process requests. 
Autoscaling at which level?